generator/common/beatgen/setup/creator.go

```diff
@@ -0,0 +1,87 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package setup
+
+import (
+	"fmt"
+	"go/build"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	devtools "github.com/elastic/beats/dev-tools/mage"
+	"github.com/pkg/errors"
+)
+
+// CfgPrefix specifies the env variable prefix used to configure the beat
+var CfgPrefix = "NEWBEAT"
```

Comment:
Why not a constant here? `CFG_PREFIX`

Reply:
I don't really see all uppercase names in go too often?

Reply:
+1 to don't use uppercase name, but I think this could still be a constant.
```suggestion
const CfgPrefix = "NEWBEAT"
```